add later versions (#16076)
dotnet/docs: docs/core/porting/project-structure.md

  - Allows you to write code for a specific .NET Framework version in your libraries through the use of compiler directives.

  *Unsupported scenarios:*

>   - Requires developers to use Visual Studio 2017 os later versions to open existing projects. To support older versions of Visual Studio, [keeping your project files in different folders](#support-vs) is a better option.

Reviewer: ```suggestion
  - Requires developers to use Visual Studio 2017 or a later version to open existing projects. To support older versions of Visual Studio, [keeping your project files in different folders](#support-vs) is a better option.
```

Author: That's a good point. I'll open a PR later to fix all the occurrences of that in our docs then:
https://github.com/dotnet/docs/search?q=%22or+later+versions%22&unscoped_q=%22or+later+versions%22

- <a name="support-vs"></a>[**Keep existing projects and new .NET Core projects separate**](#keep-existing-projects-and-create-a-net-core-project)

  *What this is good for:*

>   - Continuing to support development on existing projects without having to upgrade for developers/contributors who may not have Visual Studio 2017 or later versions.

Reviewer: ```suggestion
  - Supporting development on existing projects for developers and contributors who may not have Visual Studio 2017 or a later version.
```
